packages/components/src/Accordion/Accordion.tsx

```diff
@@ -50,9 +51,16 @@ export const Accordion: React.FC<AccordionProps> = ({
 }) => {
   const [collapsed, setCollapsed] = useState(initialOpen || false);
 
+  useEffect(() => {
```

Comment:
Ist das nicht durch den Aufruf von useState mit initialOpen überflüssig?

Reply:
Nein, eine Änderung von außen useEffect nicht berücksichtigt

Reply:
Das ist in der Tat der QuickFix, damit man den State von außen steuern kann. Ohne UseEffect würde die Komponente nicht neu rendern wenn der initialSate sich ändert 👍

Reply:
initialState soll sich ja auch eigentlich nicht ändern. Wenn's ein Quickfix ist sollte nicht untergehen, das zukünftig zu fixen.